digicampipe/utils/histogram.py

```diff
@@ -0,0 +1,52 @@
+from digicampipe.io.containers_calib import CalibrationHistogramContainer
+import numpy as np
+from histogram.histogram import Histogram1D
+
+
+def convert_histogram_to_container(histogram):
```

Comment:
I think this could also be a method of `CalibrationHistogramContainer` ... named `CalibrationHistogramContainer.from_histogram()`.

I mean this function needs to know exactly what the attributes of `CalibrationHistogramContainer` are, so when the container is modified, this function also needs to be modified. It makes sense to keep stuff which needs to be modified for the same reason close together.

Reply:
Done